go.mod

```diff
@@ -259,3 +259,8 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.14.3-0.20230601165947-6ce0bf390ce3 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+// FIXME: For Nutanix failure domain support
+replace github.com/openshift/api => github.com/nutanix-cloud-native/openshift-api v0.0.0-20231115190355-c3972acd2be6
+
+replace github.com/openshift/client-go => github.com/yanhua121/client-go v0.0.0-20231115191314-2d5bcdc083eb
```

Comment:
i'm guessing we are waiting for the other PRs to merge?

Reply:
Updated the dependency versioning.